packages/fractal/fractal/src/config/defaults.js

```diff
@@ -19,6 +19,13 @@ module.exports = {
         views: {
           match: 'view.*',
           default: '.html'
+        },
+        assets: {
+          styles: '**/*.{css,scss}',
+          scripts: '**/*.{js}',
```

Comment:
Just testing this and it seems that the `scripts` glob here won't actually match any JS files - the `{}` syntax only works if there is more than one comma separated option within it, so this should actually be `**/*.js`